### DISCUSSION TITLE
GTK 4 Installation on linux mint 19 (64 bits)

### DISCUSSION THREAD

**Discussion by gummybears** [hugopl/gtk4.cr#42] [General]
I am not able to install the GTK4 dependendies. 
```
$ sudo apt-get install libgtk-4-1 libgirepository1.0-dev gobject-introspection gir1.2-gtk-4.0
```
[sudo] password for .....               
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libgtk-4-1 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libgtk-4-1' has no installation candidate
E: Unable to locate package gir1.2-gtk-4.0
E: Couldn't find any package by glob 'gir1.2-gtk-4.0'
E: Couldn't find any package by regex 'gir1.2-gtk-4.0'
```

**Comment by hugopl** [comment]
I think it’s better to ask for help on Linux Mint forums about how to install GTK4 there. I never used Linux Mint, so I can’t help you 😞 .
